btree/btree.c

```diff
@@ -8333,31 +8333,61 @@ static int64_t btree_truncate_tick(struct m0_sm_op *smop)
 	};
 }
 
+#ifndef __KERNEL__
+static void unmap_node(void* addr, int64_t size)
+{
+	int rc;
+	rc = munmap(addr, size);
+	if (rc != 0)
+		M0_LOG(M0_ERROR, "Unmapping of memory failed");
```

Comment:
Return an error code based on which the caller will decide the next action to execute.

Reply:
Done